tests/test_intermediary.py

```diff
@@ -94,6 +94,10 @@ def test_from_github_open(self):
         self.checkResponseFields(response)
 
         self.assertEqual(response.fixVersion, ['mock_milestone'])
+        self.assertEqual(response.priority, 'mock_priority')
+        self.assertEqual(response.content, 'mock_content')
+        self.assertEqual(response.reporter, 'mock_reporter')
+        self.assertEqual(response.assignee, 'mock_assignee')
```

Comment:
Can some or all of these assertions be added to `checkResponseFields()` instead of repeating them in each test?

Reply:
Done. ce912603abb8262570d7ff20c113da839bfac19a